### ISSUE TITLE
Get nothing when using API call function

### ISSUE THREAD

**Issue by bigredbug47** [petrpatek/tripadvisor-scraper#17]
Hi guys,

I try to get reviews from 1 hotel by calling your API function, but it returns nothing. 

Here is the input and the output of my postman:

Input:
```
{
    "hotelId": "672866",
    "includeReviews": true,
    "lastReviewDate": "2018-01-12",
    "proxyConfiguration": {
	    "useApifyProxy": true,
	    "apifyProxyGroups": [
	      "GOOGLESERP"
	    ]
	}
}
```

Output:
```
{
    "data": {
        "id": "HomRtm67XkcfQsrb3",
        "actId": "C3EHyBNnFuHsT2Yn5",
        "userId": "S8q674iKJc32wjqWb",
        "startedAt": "2019-07-11T05:08:44.354Z",
        "finishedAt": null,
        "status": "READY",
        "meta": {
            "origin": "API",
            "userAgent": "PostmanRuntime/7.15.0"
        },
        "stats": {
            "inputBodyLen": 268,
            "restartCount": 0
        },
        "options": {
            "build": "latest",
            "timeoutSecs": 6000,
            "memoryMbytes": 8192,
            "diskMbytes": 16384
        },
        "buildId": "uby54XhgfFhP9w9BF",
        "defaultKeyValueStoreId": "BvbCjd5SKBRpDtFww",
        "defaultDatasetId": "wx4e2viiJLjzNWG6r",
        "defaultRequestQueueId": "9NDmT2jyfdzTk6JEM",
        "buildNumber": "0.0.46",
        "containerUrl": "https://ubu4atmwfutk.runs.apify.net"
    }
}
```

Please let me know where I am wrong, thank a lot.

**Comment by petrpatek** [comment]
HI,
Are you sending POST request to this endpoint - `https://api.apify.com/v2/acts/actorId/run-sync?`?

**Comment by petrpatek** [comment]
I found the issue should be fixed in #19.
